File: TechModule/SoftwareTechnologies/ExamPreps/Shopping List/PHP Solution/src/AppBundle/Entity/Product.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints\NotBlank;

/**
 * Product
 *
 * @ORM\Table(name="products")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\ProductRepository")
 */
class Product
{
    /**
     * @var int
     *
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ORM\Column(type="integer")
     */
    private $id;
    /**
     * @var string
     *
     * @ORM\Column(type="string", length=255)
     * @NotBlank
     */
    private $name;
    /**
     * @var string
     *
     * @ORM\Column(type="string", length=255)
     * @NotBlank
     */
    private $status;

    /**
     * @var int
     *
     * @ORM\Column(type="integer")
     * @NotBlank
     */
    private $quantity;

    /**
     * @var int
     *
     * @ORM\Column(type="integer")
     * @NotBlank
     */
    private $priority;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     * @return Product
     */
    public function setId($id)
    {
        $this->id = $id;
        return $this;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $name
     * @return Product
     */
    public function setName($name)
    {
        $this->name = $name;
        return $this;
    }

    /**
     * @return string
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * @param string $status
     * @return Product
     */
    public function setStatus($status)
    {
        $this->status = $status;
        return $this;
    }

    /**
     * @return int
     */
    public function getQuantity()
    {
        return $this->quantity;
    }

    /**
     * @param int $quantity
     * @return Product
     */
    public function setQuantity($quantity)
    {
        $this->quantity = $quantity;
        return $this;
    }

    /**
     * @return int
     */
    public function getPriority()
    {
        return $this->priority;
    }

    /**
     * @param int $priority
     * @return Product
     */
    public function setPriority($priority)
    {
        $this->priority = $priority;
        return $this;
    }



}

File: Tech-Module/Software-Technologies/Shopping List/Java Solution/src/main/java/shoppinglist/entity/Product.java
package shoppinglist.entity;

import javax.persistence.*;

@Entity
@Table(name = "products")
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(nullable = false)
    private String Name;

    @Column(nullable = false)
    private Integer Priority;

    @Column(nullable = false)
    private Integer Quantity;

    @Column(nullable = false)
    private String Status;

    public int getId() {
        return id;
    }

    public Product setId(int id) {
        this.id = id;
        return this;
    }

    public String getName() {
        return Name;
    }

    public Product setName(String name) {
        Name = name;
        return this;
    }

    public Integer getPriority() {
        return Priority;
    }

    public Product setPriority(Integer priority) {
        Priority = priority;
        return this;
    }

    public Integer getQuantity() {
        return Quantity;
    }

    public Product setQuantity(Integer quantity) {
        Quantity = quantity;
        return this;
    }

    public String getStatus() {
        return Status;
    }

    public Product setStatus(String status) {
        Status = status;
        return this;
    }
}
